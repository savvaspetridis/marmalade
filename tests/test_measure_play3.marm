/*
Testing measure play:	indexing from phrase
*/


measure a_1 = $(4:4) [47.e, 0.e, 47.e, 0.e, 47.e, 0.e, 47.e, 0.e];
measure a_2 = $(4:4) [52.e, 0.e, 52.e, 0.e, 52.e, 0.e, 52.e, 0.e];
measure a_3 = $(4:4) [0.w];

phrase ph_1 = $(BASS) [a_1, a_2, a_3];

measure swag = ph_1&1;

(play()) [swag];