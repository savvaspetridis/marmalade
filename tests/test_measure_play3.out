jMusic Play: Playing score Untitled Phrase copy copy score using JavaSound General MIDI soundbank.
jMusic Play: Waiting for the end of Untitled Phrase copy copy score.